electron/install.js

```diff
@@ -5,25 +5,24 @@ const sudo = require('sudo-prompt');
 const process = require('process');
 const axios = require('axios');
 const { spawnSync } = require('child_process');
+const { logger } = require('./logger');
 
 const { paths } = require('./constants');
 
-/**
- * current version of the pearl release
- * - use "" (nothing as a suffix) for latest release candidate, for example "0.1.0rc26"
- * - use "alpha" for alpha release, for example "0.1.0rc26-alpha"
- */
-const OlasMiddlewareVersion = '0.1.0rc71';
+/** @deprecated PyPi is not used any more, this OlasMiddlewareVersion will be removed in the future */
```

Comment:
I may be wrong on this comment, will leave as is for now.. Haven't run through the places it's used properly yet!
```suggestion
/**
 * current version of the pearl release
 * - use "" (nothing as a suffix) for latest release candidate, for example "0.1.0rc26"
 * - use "alpha" for alpha release, for example "0.1.0rc26-alpha"
 */
```

Reply:
@truemiller Please feel free to commit the suggestion on the comment if necessary.